File: src/Python/Parcial2/Geometria/circunferencia.py
import math
from poligono import Poligono

class Circunferencia(Poligono):

    def __init__(self, numLados, radio):
        super().__init__(numLados)
        self._radio = radio

    def calcularArea(self):
        return math.pi * self._radio ** 2

    def calcularPerimetro(self):
        return 2 * math.pi * self._radio

File: src/C#/Parcial2/Geometria/Circunferencia.cs
using System;

public class Circunferencia : Poligono
{
    protected double radio;

    public Circunferencia(int numLados, double radio)
    {
        this.numLados = numLados;
        this.radio = radio;
    }

    public override double calcularArea()
    {
        return Math.PI * radio * radio;
    }

    public override double calcularPerimetro()
    {
        return 2 * Math.PI * radio;
    }
}